client-common/src/main/java/org/apache/cassandra/sidecar/common/request/data/RestoreJobProgressRequestParams.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.request.data;
+
+import java.util.UUID;
+
+import org.apache.cassandra.sidecar.common.data.RestoreJobProgressFetchPolicy;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Parameters to fetch the restore job progress
+ */
+public class RestoreJobProgressRequestParams
+{
+    public final String keyspace;
+    public final String table;
```

Comment:
QualifiedName?

Reply:
It does not seem to be necessary, although it would be valid to wrap them in `QualifiedName`.

The values are used in `org.apache.cassandra.sidecar.common.request.RestoreJobProgressRequest` mainly. Other request object take string value for keyspace and table too.